examples/compare_connectivity_over_time_over_trial.py

```diff
@@ -0,0 +1,392 @@
+n_epochs = 5 # number of simulated epochs
+n_channels = 3 # number of channels
+n_times = 2000 # number of sample points
+sfreq = 250 # # Set sampling freq
+data = np.random.rand(n_epochs, n_channels, n_times) # generate random data
+
+# In case 1, we overwrite all epochs with the data from the first epoch
+for i in range(n_epochs):
+    data[i] = data[0]
+
+ch_names = ["C3","Cz","C4"] # three random channel names
+info = mne.create_info(ch_names, sfreq, ch_types="eeg") # create info object
+data_epoch = mne.EpochsArray(data,info) # create EpochsArray
+
+data_epoch.plot(scalings=1) # Visualize the data
+
+###############################################################################
+# First we compute connectivity over trials.
+
+# Freq bands of interest
+Freq_Bands = {"theta": [4.0, 8.0], 
+              "alpha": [8.0, 13.0],
+              "beta": [13.0, 30.0]}
+n_freq_bands = len(Freq_Bands)
+min_freq = np.min(list(Freq_Bands.values()))
+max_freq = np.max(list(Freq_Bands.values()))
+
+# Provide the freq points
+freqs = np.linspace(min_freq,max_freq,int((max_freq-min_freq)*4+1))
+
+# The dictionary with frequencies are converted to tuples for the function
+fmin=tuple([list(Freq_Bands.values())[f][0] for f in range(len(Freq_Bands))])
+fmax=tuple([list(Freq_Bands.values())[f][1] for f in range(len(Freq_Bands))])
+
+# We specify the connectivity measurements
+connectivity_methods = ["plv", "wpli"]
+n_con_methods=len(connectivity_methods)
+
+# Pre-allocatate memory for the connectiviy matrices
+con_epochs_array = np.zeros((n_con_methods, n_channels, n_channels,
+                             n_freq_bands, n_times))
+con_epochs_array[con_epochs_array==0] = np.nan # nan matrix as 0 is meaningful
+
+# Compute connectivity over trials
+con_epochs = spectral_connectivity_epochs(data_epoch,
+                                          method=connectivity_methods,
+                                          sfreq=sfreq, mode="cwt_morlet",
+                                          cwt_freqs=freqs, fmin=fmin,
+                                          fmax=fmax, faverage=True)
+
+# Get data as connectivity matrices
+for c in range(n_con_methods):
+    con_epochs_array[c] = con_epochs[c].get_data(output="dense")
+
+###############################################################################
+# As previously mentioned, connectivity over trials can give connectivity
+# for each timepoint, here in the form of 
+# :class:`mne_connectivity.SpectroTemporalConnectivity`.
+# However, in this example we are not interested in the timing, so we will 
+# average over all timepoints. Notice that only ``mode="cwt_morlet"`` will
+# return an instance of :class:`mne_connectivity.SpectroTemporalConnectivity`
+# and ``mode="fourier"`` or ``mode="multitaper"`` returns an instance of 
+# :class:`mne_connectivity.SpectralConnectivity`, which does not have
+# single timepoint resolution.
+
+con_epochs_array = np.mean(con_epochs_array, axis=4) # average over timepoints
```

Comment:
If we want to average `SpectroTemporalConnectivity` container over time, can we use the `combine` method? https://mne.tools/mne-connectivity/dev/generated/mne_connectivity.SpectroTemporalConnectivity.html#mne_connectivity.SpectroTemporalConnectivity.combine

Reply:
I just tried, but combine only works for averaging over epochs, which ``spectral_connectivity_epochs`` already do if I understood the ``nave`` correctly (when printing SpectroTemporalConnectivity).